### PR TITLE
add python 3 job to jenkinsfile

### PR DESCRIPTION
**Pull request by estute**
Parameterize the jenkinsfile so that it can run unit tests w/ different tox/sandbox environments

### CODE REVIEW

**Comment by estute** [comment]
jenkins test python 2

**Comment by estute** [comment]
jenkins test python 3

**Comment by estute** [comment]
jenkins test python 2

File: Jenkinsfile [hunk 1]
```diff
@@ -18,17 +18,17 @@ pipeline {
 
         stage('Run the tests for each sandbox') {
             parallel {
-                stage('Run tests with python 2.7 without proxy') {
+                stage('Run unit tests without proxy') {
                     environment {
                         CODEJAIL_TEST_USER = 'sandbox'
-                        CODEJAIL_TEST_VENV = '/home/sandbox/codejail_sandbox-python2.7'
+                        CODEJAIL_TEST_VENV = "/home/sandbox/codejail_sandbox-python${PYTHON_VERSION}"
```
Comment (feanil) [inline Jenkinsfile]: Is there a default for ${PYTHON_VERSION} that's set elsewhere?

Comment (feanil) [reply Jenkinsfile]: Nevermind, I see you other PR.

**Pull request merged**